carla_integration/docker-compose.yml

```diff
@@ -58,4 +58,17 @@ services:
     volumes:
       - /opt/carma/logs:/opt/carma/logs
       - /opt/carma/.ros:/home/carma/.ros
-    command: bash -c 'export ROS_NAMESPACE=$${CARMA_INTR_NS} && wait-for-it.sh localhost:11311 -- roslaunch /opt/carma/vehicle/config/drivers.launch drivers:=carla_ackermann_control_wrapper'
\ No newline at end of file
+    command: bash -c 'export ROS_NAMESPACE=$${CARMA_INTR_NS} && wait-for-it.sh localhost:11311 -- roslaunch /opt/carma/vehicle/config/drivers.launch drivers:=carla_ackermann_control_wrapper'
```

Comment:
you need to remove the drivers:=carla_ackermann_control_wrapper
It will error out since the launchfile has no more arguments.